frontend/src/components/Main/GridViewCoupons.tsx

```diff
@@ -1,12 +1,12 @@
 import styled from '@emotion/styled';
 import { Coupon } from '../../types';
-import GridViewCoupon from './GridViewCoupon';
+import MainCoupon from './MainCoupon';
 
 const GridViewCoupons = ({ coupons }: { coupons: Coupon[] }) => {
   return (
     <S.Container>
       {coupons.map(coupon => (
-        <GridViewCoupon key={coupon.couponHistoryId} coupon={coupon} />
+        <MainCoupon coupon={coupon} />
```

Comment:
```suggestion
        <ModalWrapper >
          <GridviewCoupon key= {}>
         <ModalWrapper />
```

처럼 싸주는게 더 나을 거 같아요 나중에도 Modal Wrapper안에 children 박을 수 있으니까요